### PR TITLE
document for HTMLAnchorElement.ping

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
This PR add documentation for HTMLAnchorElement.ping property.

It is part of the discussion to document all interoperable features ([mdn/discussions#476](https://github.com/orgs/mdn/discussions/476))
### Resource
[ping](https://html.spec.whatwg.org/multipage/text-level-semantics.html#dom-a-ping)

### CODE REVIEW

File: files/en-us/web/api/htmlanchorelement/ping/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: "HTMLAnchorElement: ping property"
+short-title: ping
+slug: Web/API/HTMLAnchorElement/ping
+page-type: web-api-instance-property
+browser-compat: api.HTMLAnchorElement.ping
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`ping`** property of the {{domxref("HTMLAnchorElement")}} interface is a space-separated list of URLs.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlanchorelement/ping/index.md:11]:
```suggestion
The **`ping`** property of the {{domxref("HTMLAnchorElement")}} interface is a space-separated list of URLs that will be used to notify if the user follows the hyperlink.
```

the properties list also need update

File: files/en-us/web/api/htmlanchorelement/ping/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+> **Note:** This property is not effective in Firefox and its usage may be limited due to privacy and security concerns.
+
+## Example
+
+```html
+<a
+  id="exampleLink"
+  href="https://example.com"
+  ping="https://example-tracking.com"
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlanchorelement/ping/index.md:23]: may we update the example for the case of many urls, like `ping="https://example-tracking.com https://example-tracking.com"`

File: files/en-us/web/api/htmlanchorelement/index.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
   - : A string containing the password specified before the domain name.
 - {{domxref("HTMLAnchorElement.pathname")}}
   - : A string containing an initial `'/'` followed by the path of the URL, not including the query string or fragment.
+- {{domxref("HTMLAnchorElement.ping")}}
+  - : A space-separated list of URLs. When the link is followed, the browser will send POST requests with the body PING to the URLs.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlanchorelement/index.md:37]:
```suggestion
- {{domxref("HTMLAnchorElement.ping")}}
  - : A space-separated list of URLs. When the link is followed, the browser will send {{HTTPMethod("POST")}} requests with the body PING to the URLs.
```

File: files/en-us/web/api/htmlanchorelement/ping/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+title: "HTMLAnchorElement: ping property"
+short-title: ping
+slug: Web/API/HTMLAnchorElement/ping
+page-type: web-api-instance-property
+browser-compat: api.HTMLAnchorElement.ping
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`ping`** property of the {{domxref("HTMLAnchorElement")}} interface is a space-separated list of URLs. When the link is followed, the browser will send POST requests with the body PING to the URLs.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmlanchorelement/ping/index.md:11]:
```suggestion
The **`ping`** property of the {{domxref("HTMLAnchorElement")}} interface is a space-separated list of URLs. When the link is followed, the browser will send {{HTTPMethod("POST")}} requests with the body PING to the URLs.
```

Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlanchorelement/ping/index.md]: Thank you @skyclouds2001

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**